docs/basic/proxy.mdx

```diff
@@ -6,6 +6,8 @@ description: 'Create a proxy object.'
 
 # proxy
 
+The proxy tracks changes to the original object and all nested objects, notifying listeners when a new version of the object is created.
```

Comment:
> created

Precisely speaking, object doesn't have to be created, but just modified. Like `++state.count` doesn't create a new object.